tests/src/testPathFollowing.cpp

```diff
@@ -154,6 +152,43 @@ auto simple2DOperatorDisplacementControlTest() {
   const auto controlInfo = dc.run();
 
   TestSuite t("Displacement Control with Subsidiary function");
+  t.check(controlInfo.success, "No convergence");
```

Comment:
All the test here should not only check for sucess, but also for more. (steps used and intermediated values, since if in the future someone needs to refactor or add things, the behaviour could change with any warning